Azure Container Apps session execute new api-version (#8249)
Azure/azure-cli-extensions: src/containerapp/HISTORY.rst

* 'az containerapp create': Fix Role assignment error when the default Azure Container Registry could not be found
* Upgrade api-version to 2024-10-02-preview
* 'az containerapp create/update': `--yaml` support property pollingInterval and cooldownPeriod

> * 'az containerapp session code-interpreter': `--path` support upload-file/list-files/show-file-content/show-file-metadata/delete-file for code interpreter sessions

Reviewer: Hi @bowen5 @yitaopan 
About the request payload change and the response breaking change, we should update the description about them

* 'az containerapp create': Fix Role assignment error when the default Azure Container Registry could not be found
* Upgrade api-version to 2024-10-02-preview
* 'az containerapp create/update': `--yaml` support property pollingInterval and cooldownPeriod

> * 'az containerapp session code-interpreter': `--path` support upload-file/list-files/show-file-content/show-file-metadata/delete-file for code interpreter sessions

Reviewer: ```suggestion
* 'az containerapp session code-interpreter upload-file/list-files/show-file-content/show-file-metadata/delete-file': Support `--path` to specify the path of code interpreter session
```

Author: Fixed

> * 'az containerapp session code-interpreter': Update response payload format for api-version 2024-10-02-preview

Reviewer: ```suggestion
* [Breaking Change] 'az containerapp session code-interpreter': Update response payload format for api-version 2024-10-02-preview
```
